docs/apis/http/tus_upload.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "TUS upload"
```

Comment:
```suggestion
title: "tus Upload"
```

Reply:
This is how tus is spelled on https://tus.io/ so I guess, we should use that spelling, even at the beginning of sentences.